### DISCUSSION TITLE
Release 0.7

### DISCUSSION THREAD

**Discussion by N4S4** [N4S4/synology-api#149] [Announcements]
## What's Changed
* Added Surveillance Station by @N4S4 
* Added possibility to use DownloadStation2 api if required by @N4S4 
* rename_folder function in filestation optimization by @N4S4 
* Feature/implement entry request api by @FBoissadier in https://github.com/N4S4/synology-api/pull/137
* Fix mime types by @bchaintreuil in https://github.com/N4S4/synology-api/pull/139
* New get_file serve by @cardenastw in https://github.com/N4S4/synology-api/pull/140
* added backup_task_list by @insanity67 in https://github.com/N4S4/synology-api/pull/142
* Added list_storage method for SYNO.ActiveBackup.Share by @BamButz in https://github.com/N4S4/synology-api/pull/143
* Login_out standardized by @N4S4 in https://github.com/N4S4/synology-api/pull/146
* add clieantlist & reservations for DHCPServer by @marcelrv in https://github.com/N4S4/synology-api/pull/147
* update readme by @N4S4 in https://github.com/N4S4/synology-api/pull/148

## New Contributors
* @FBoissadier made their first contribution in https://github.com/N4S4/synology-api/pull/137
* @bchaintreuil made their first contribution in https://github.com/N4S4/synology-api/pull/139
* @cardenastw made their first contribution in https://github.com/N4S4/synology-api/pull/140
* @insanity67 made their first contribution in https://github.com/N4S4/synology-api/pull/142
* @BamButz made their first contribution in https://github.com/N4S4/synology-api/pull/143
* @marcelrv made their first contribution in https://github.com/N4S4/synology-api/pull/147

**Full Changelog**: https://github.com/N4S4/synology-api/compare/v0.6.3...v0.7

<hr /><em>This discussion was created from the release <a href='https://github.com/N4S4/synology-api/releases/tag/v0.7'>Release 0.7</a>.</em>

**Comment by ghost** [comment]
File upload not working. Failed login - error 119 SID not found. With the same FileStation session other methods works.

**Reply by N4S4** [reply]
I will check if i can replicate the issue

**Reply by N4S4** [reply]
that's odd because sid is present. It does not look like the issue is in the code, trying to look for a cause.

**Reply by N4S4** [reply]
Please update to v0.7.1 for the issue to be fixed
